src/Sentry.Serilog/SentrySinkExtensions.cs

```diff
@@ -144,6 +144,112 @@ public static class SentrySinkExtensions
+        /// <param name="reportAssemblies">Whether or not to include referenced assemblies in each event sent to sentry. Defaults to <see langword="true"/>. <seealso cref="SentryOptions.ReportAssemblies"/></param>
+        /// <param name="deduplicateMode">What modes to use for event automatic de-duplication. <seealso cref="SentryOptions.DeduplicateMode"/></param>
+        /// <param name="initializeSdk">Whether to initialize this SDK through this integration. <seealso cref="SentrySerilogOptions.InitializeSdk"/></param>
+        /// <param name="defaultTags">Defaults tags to add to all events. <seealso cref="SentryOptions.DefaultTags"/></param>
+        /// <returns><see cref="LoggerConfiguration"/></returns>
+        /// <example>This sample shows how each item may be set from within a configuration file:
+        /// <code>
+        /// {
+        ///     "Serilog": {
+        ///         "Using": [
+        ///             "Serilog",
+        ///             "Sentry",
+        ///         ],
+        ///         "WriteTo": [{
+        ///                 "Name": "Sentry",
+        ///                 "Args": {
+        ///                     "dsn": "https://MY-DSN@sentry.io",
+        ///                     "minimumBreadcrumbLevel": "Verbose",
+        ///                     "minimumEventLevel": "Error",
+        ///                     "outputTemplate": "{Timestamp:o} [{Level:u3}] ({Application}/{MachineName}/{ThreadId}) {Message}{NewLine}{Exception}"///
+        ///                     "sendDefaultPii": false,
+        ///                     "isEnvironmentUser": false,
+        ///                     "serverName": "MyServerName",
+        ///                     "attachStackTrace": false,
+        ///                     "maxBreadcrumbs": 20,
+        ///                     "sampleRate": 0.5,
+        ///                     "release": "0.0.1",
+        ///                     "environment": "staging",
+        ///                     "maxQueueItems": 100,
+        ///                     "shutdownTimeout": "00:00:05",
+        ///                     "decompressionMethods": "GZip",
+        ///                     "requestBodyCompressionLevel": "NoCompression",
+        ///                     "requestBodyCompressionBuffered": false,
+        ///                     "debug": false,
+        ///                     "diagnosticLevel": "Debug",
+        ///                     "reportAssemblies": false,
+        ///                     "deduplicateMode": "All",
+        ///                     "initializeSdk": true,
+        ///                     "defaultTags": {
+        ///                         "key-1", "value-1",
+        ///                         "key-2", "value-2"
+        ///                     }
+        ///                 }
+        ///             }
+        ///         ]
+        ///     }
+        /// }
+        /// </code>
+        /// </example>
+        public static LoggerConfiguration Sentry(
```

Comment:
Can we add an obsolete to remove on the next major? Will require supressing warnings of the usage from tests
```suggestion
        [Obsolete("This method will be deleted on the next major. Please use the overload.")]
        public static LoggerConfiguration Sentry(
```